ifao-publications.csl

```diff
@@ -0,0 +1,335 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+  <info>
+    <title>IFAO publications</title>
+    <id>http://www.zotero.org/styles/ifao-publications</id>
+    <link href="http://www.zotero.org/styles/ifao-publications" rel="self"/>
+    <link href="http://www.zotero.org/styles/gallia" rel="template"/>
+    <link href="https://www.ifao.egnet.net/publications/publier/normes-ed/" rel="documentation"/>
+    <author>
+      <name>Alexandre Vincent</name>
+      <email>alexandre.vincent@univ-poitiers.fr</email>
+    </author>
+    <author>
+      <name>Nicolas Monteix</name>
+      <email>nicolas.monteix@univ-rouen.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <category field="anthropology"/>
+    <summary>Author-date style for IFAO series</summary>
+    <updated>2020-04-24T10:09:48+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor" form="short">éd.</term>
+      <term name="accessed">consulté le</term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <style-options punctuation-in-quote="false"/>
+    <terms>
+      <term name="translator" form="short">trans.</term>
+      <term name="open-quote">“</term>
```

Comment:
You should also define the inner quotes then (just copy from the en-US locale)

Reply:
Done